### DISCUSSION TITLE
#51 My model is not trained

### DISCUSSION THREAD

**Discussion by niktheod** [mrdbourke/pytorch-deep-learning#323] [Q&A]
At 7:40:00 when Daniel said that we could try to recreate the model ourselves I tried to do a slightly more difficult one but it doesnt work. It wont get trained. Here is the link: https://colab.research.google.com/drive/1rAzfeJg32InHhP2pmyQe8o6n4EsUBF60#scrollTo=VPhnQiwFbjbA

Im pretty sure that Im missing something obvious but I really cant figure out what exactly. Any help will be appreciated.

**Comment by pusapatiakhilraju** [comment]
Hey NikTheod,

Can you grant access to the workbook?

**Reply by niktheod** [reply]
Did it.

**Reply by pusapatiakhilraju** [reply]
Okay, I have just checked your code and there is nothing wrong with it. However there are a couple of things i would like you to note:
1. 40 data points may be a little too less for deep learning data.
2. Your complicated data is not a straight line (it's more of an exponential graph). The architecture of the model you have built, does not have activation layers, more number of Neurons, more layers etc that can help in better predictions. The handling and prediction of such datasets is mentioned in the future videos by Daniel.  

Once you learn new techniques in future videos, come back to this dataset and change the architecture of your model to include non linear activation functions, fine tune the parameters like (epochs, Nuerons etc) and this will help you achieve better results.

Hoping this clarifies your question.
